### PR TITLE
4 static product page

### LINKED ISSUE #4: Static product page

### PR DESCRIPTION
**Pull request by Michael-Dev51**
close #4

### CODE REVIEW

File: assets/scss/abstracts/_variables.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 $background:#F2Deee;
 $backgroundPrice:#915cf4;
 $descriptive: #ede6eb;
-$price:#fff;
\ No newline at end of file
+$price:#fff;
+$btnCart: #eface2;
+$border:#ffc0cb;
```
Comment (Tutt-J) [inline assets/scss/abstracts/_variables.scss]: Please name your color like grey, black, pink, etc...

File: pages/product.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="../assets/dist/css/product.css"
+    />
+    <script
+      defer
+      src="https://kit.fontawesome.com/ca4c042e28.js"
+      crossorigin="anonymous"
+    ></script>
```
Comment (Tutt-J) [inline pages/product.html:53]: Missing og and twitter

File: assets/scss/pages/product.scss [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        max-width: 100%;
+      }
+      span {
+        font-weight: bold;
+        margin-top: 15px;
+      }
+    }
+  }
+}
+#customization {
```
Comment (Tutt-J) [inline assets/scss/pages/product.scss]: Must be inside product

File: assets/scss/pages/product.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      flex-basis: 40%;
+    }
+    select {
+      flex-basis: 50%;
+    }
+  }
+  #addToCart {
+    width: 50%;
+    margin-left: 175px;
+  }
```
Comment (Tutt-J) [inline assets/scss/pages/product.scss]: Must be inside product

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: assets/scss/pages/product.scss (hunk 4 above)
Comment (Michael-Dev51) [reply assets/scss/pages/product.scss]: It is done

File: assets/scss/pages/product.scss (hunk 3 above)
Comment (Michael-Dev51) [reply assets/scss/pages/product.scss]: It is done

File: assets/scss/abstracts/_variables.scss (hunk 1 above)
Comment (Michael-Dev51) [reply assets/scss/abstracts/_variables.scss]: $darkPink: #eface2;
$pink:#ffc0cb;

File: pages/product.html (hunk 2 above)
Comment (Michael-Dev51) [reply pages/product.html:53]: <meta property="og:type" content="website" />
    <meta
      property="og:url"
      content="https://orinoco.michael-blondeau.fr/pages/product.html"
    />
    <meta property="og:title" content="Orinoco" />
    <meta
      property="og:description"
      content="Orinoco est un site e-commerce à thème."
    />
    <meta
      property="og:image"
      content="http://orinoco.michael-blondeau.fr/assets/img/logo.png"
    />
    
    <meta property="twitter:card" content="summary_large_image" />
    <meta
      property="twitter:url"
      content="https://orinoco.michael-blondeau.fr/pages/product.html"
    />
    <meta property="twitter:title" content="Orinoco" />
    <meta
      property="twitter:description"
      content="Orinoco est un site e-commerce à thème."
    />
    <meta
      property="twitter:image"
      content="http://orinoco.michael-blondeau.fr/assets/img/logo.png"
    />

**Pull request merged**